src/utils/__tests__/get-code-from-ast.spec.ts

```diff
@@ -30,6 +30,7 @@ import a from 'a';
     expect(format(formatted, { parser: 'babel' })).toEqual(
         `// first comment
 // second comment
+
```

Comment:
I don't see the point adding new lines ? Is this coming from an IDE formater ?

Reply:
no, for some reason the snapshot doesn't match with those new line

Reply:
They are generated after `@babel/generator` >=7.19.

Reply:
@byara It seems that they are auto-generated by `babel/generator`

Reply:
With 7.19.3 the tests pass, but 7.19.4 they fail. Probably due to this PR: https://github.com/babel/babel/pull/14979

Reply:
@Eldemarkki The PR tagged as 8.0.0-alpha.2 but the changelog saids 7.19.4.
Did they just messed up the release?

Reply:
seems its an unvoidable breaking change